web/src/pages/Cases/CaseDetails/Voting/VotesDetails/index.tsx

```diff
@@ -41,13 +45,13 @@ const StyledCard = styled(Card)`
   width: 100%;
   height: auto;
   padding: 11.5px ${responsiveSize(8, 18)};
-  margin: 8px 0;
+  border: 1px solid ${({ theme }) => theme.stroke};
```

Comment:
Removing the top margin looks good, but removing all margins feels less polished I think.
![image](https://github.com/user-attachments/assets/b0e38685-7173-46f4-9fd5-183e0e383916)
![image](https://github.com/user-attachments/assets/5ee0fbd7-347f-4d93-8b4e-25b77a1bb38a)